simphony/cuds/abc_modeling_engine.py

```diff
@@ -22,6 +25,31 @@ class ABCModelingEngine(object):  # pragma: no cover
     """
     __metaclass__ = ABCMeta
 
+    def __init__(self, *args, **kwargs):
```

Comment:
Is the flexibility in the __init__ a concern for interoperability?  (think the "use_internal_interface" switch in lammps)